src/vs/platform/windows/electron-main/windowImpl.ts

```diff
@@ -958,6 +972,23 @@ export class CodeWindow extends BaseWindow implements ICodeWindow {
 		}
 	}
 
+	private async startCollectingJScallStacks(): Promise<void> {
+		if (this.JScallStackCollector) {
+			return;
+		}
+		this.JScallStackCollector = setInterval(async () => {
+			const stack = await this._win.webContents.mainFrame.collectJavaScriptCallStack();
+			this.logService.error('CodeWindow unresponsive : ', stack);
```

Comment:
Should we remember the last printed stack and simply ignore it when it always repeats?